src/main/java/edu/cornell/kfs/concur/batch/ConcurRequestExtractCreatePdpFeedStep.java

```diff
@@ -0,0 +1,27 @@
+package edu.cornell.kfs.concur.batch;
+
+import java.util.Date;
+import org.kuali.kfs.sys.batch.AbstractStep;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractCreatePdpFeedService;
+
+public class ConcurRequestExtractCreatePdpFeedStep extends AbstractStep {
+	
+    private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurRequestExtractCreatePdpFeedStep.class);
+    protected ConcurRequestExtractCreatePdpFeedService concurRequestExtractCreatePdpFeedService;
+
+    @Override
+    public boolean execute(String jobName, Date jobRunDate) throws InterruptedException {
+        boolean jobCompletedSuccessfully = true;
+		
+        concurRequestExtractCreatePdpFeedService.createPdpFeedsFromRequestExtracts();
+        return jobCompletedSuccessfully;
```

Comment:
Since the createPdpFeeds method does not return a boolean is there a case that could change the value of jobCompletedSuccessfully to false? I wonder if this should just return true.